### PR TITLE
Import Provider if processing is included

### PR DESCRIPTION
**Pull request by JuhoErvasti**
Resolves: #53

### CODE REVIEW

File: {{cookiecutter.project_directory}}/plugin_templates/plugin_with_submodule.py.j2 [hunk 1]
```diff
@@ -10,8 +10,10 @@ from qgis.PyQt.QtGui import QIcon
 from qgis.PyQt.QtWidgets import QAction, QWidget
 from qgis.utils import iface
 
-{% if cookiecutter.include_processing -%}
-{% endif -%}
+{%- if cookiecutter.include_processing %}
+
+from {{cookiecutter.plugin_package}}.{{cookiecutter.plugin_package}}_processing.provider import Provider
+{%- endif %}
```
Comment (LKajan) [inline {{cookiecutter.project_directory}}/plugin_templates/plugin_with_submodule.py.j2:16]: I think those whitespace controls were correct. Did you have a reason to change those? I think that just adding the missing import between the original if block would make the ci pipeline to pass.
```suggestion
{% if cookiecutter.include_processing -%}
from {{cookiecutter.plugin_package}}.{{cookiecutter.plugin_package}}_processing.provider import Provider
{% endif -%}
```

**Review by LKajan — CHANGES_REQUESTED** [review]
Good find!
Jinja is now rendering just few extra new lines.

File: {{cookiecutter.project_directory}}/plugin_templates/plugin_with_submodule.py.j2 (hunk 1 above)
Comment (JuhoErvasti) [reply {{cookiecutter.project_directory}}/plugin_templates/plugin_with_submodule.py.j2]: I don't think there was any particular reason for that, just carelessness as I did this a bit hastily 😅

**Comment by LKajan** [comment]
#53 already resolved by #51

**Pull request closed without merging**